### DISCUSSION TITLE
Reviewers cannot make inline comments

### DISCUSSION THREAD

**Discussion by garethbrinn** [GitbookIO/community#727] [0. General bugs and questions]
Hello,

I have been informed by one of the reviewers of my work that he cannot make inline comments. I have him set an "Editor". So, he should be able to make inline comments. 

I am not sure what the issue is beyond permissions. 

Could you investigate why this may be the case? 

Kind regards,
Gareth

**Comment by aikoii** [comment]
Hey @garethbrinn! 

I couldn't replicate the issue 🤔. Could you share the CR #, space ID, and the reviewer's email/ID (you can reply here or to support@gitbook.com)?

One thing to note: currently, inline commenting isn't supported on hint and quote blocks. We have an internal ticket for this.
